lib/model.js

```diff
@@ -362,6 +366,13 @@ ModelBaseClass.prototype._initProperties = function(data, options) {
   this.trigger('initialize');
 };
 
+// Implementation of ignoreMatchedDefault property
+function ignoresMatchedDefault(property) {
+  if (property && property.ignoreMatchedDefault) {
```

Comment:
I find the option name a bit difficult to understand. "ignore" can be interpreted in many different ways.

How about reworking `ignoreMatchedDefault: true` to `persistDefaultValues: false`?

```suggestion
  if (property && property.persistDefaultValues === false) {
```

See also the existing model-level property `persistUndefinedAsNull` that's documented in https://loopback.io/doc/en/lb3/Synchronization.html#enable-change-tracking.

Reply:
Model-level  and datasource-level config are good suggestions. We'll still need the ability to exclude some properties from the blanket setting, so property-level configurability is required anyway.

Reply:
Sure. Our usual approach to this problem is to allow the flag to be set at multiple levels, where model-level flag overrides datasource-level settings and propery-level flag overrides model/datasource levels.